tests/unit/test_upstream.py

```diff
@@ -31,8 +34,12 @@
 
 @pytest.fixture
 def package_config_mock():
-    mock = flexmock(synced_files=None)
-    return mock
+    def package_config_mock_factory(upstream_package_name="test_package_name"):
+        mock = flexmock(synced_files=None, upstream_package_name=upstream_package_name)
+        mock.should_receive("current_version_command")
+        return mock
```

Comment:
You can use `mock` method to get the original `flexmock` instance:

```suggestion
        return flexmock(
            synced_files=None,
            upstream_package_name=upstream_package_name
        ).should_receive("current_version_command").mock()
```

Reply:
fixed